setup.py

```diff
@@ -49,10 +49,11 @@
         'redis==2.10.6',
         'seaborn==0.8.1',
         'uuid==1.30',
-        'pynwb>=1.0.3',
+        'pynwb==1.0.3',
```

Comment:
Hi @afonsobspinto I think we should avoif al these == here and leave as a last chance. Have you also tried with pynwb = 1.0.3 and removing hdmf? That hdmf was a bug in pynwb that they may have solved